README.md

```diff
@@ -1 +1,31 @@
-# FusionBluetooth
\ No newline at end of file
+# FusionBluetooth
The FusionBluetooth SPM package makes it possible to use Bluetooth functionality on Android and iOS using Swift 

Discuss
-------
Join our slack channel here for Fusion Package discussion [link](https://scadeio.slack.com/archives/C025WRG18TW)

For native cross plaform development with Swift and geneel Fusion introduciton, go here [SCADE Fusion](beta.scade.io/fusion)

Install - Add to Package.swift
------------------------------
```swift
import PackageDescription
import Foundation

let SCADE_SDK = ProcessInfo.processInfo.environment["SCADE_SDK"] ?? ""

let package = Package(
    name: "BluetoothApp",
    platforms: [
        .macOS(.v10_14)
    ],
    products: [
        .library(
            name: "BluetoothApp",
            type: .static,
            targets: [
                "BluetoothApp"
            ]
        )
    ],
    dependencies: [
		.package(name: "FusionBluetooth", url: "https://github.com/scade-platform/FusionBluetooth.git", .branch("main")),
    ],
    targets: [
        .target(
            name: "BluetoothApp",
            dependencies: [
            	.product(name: "FusionBluetooth", package: "FusionBluetooth"),
            ],
            exclude: ["main.page"],
            swiftSettings: [
                .unsafeFlags(["-F", SCADE_SDK], .when(platforms: [.macOS, .iOS])),
                .unsafeFlags(["-I", "\(SCADE_SDK)/include"], .when(platforms: [.android])),
            ]
        )
    ]
)
```

Permission Settings
-------------------
<Add Permission specific text and instructions>

```yaml
...
ios:
  ...
  plist:
    ...
    - key: NSBluetoothAlwaysUsageDescription
      type: string
      value: Use Bluetooth    

android:
  ...
  permissions: ["ACCESS_FINE_LOCATION", "BLUETOOTH", "BLUETOOTH_ADMIN", "ACCESS_COARSE_LOCATION"]
  ...
```

Demo App
--------
Our demo app is available here [link](https://github.com/scade-platform/FusionExamples/tree/main/BluetoothApp)


Basic Usage
-----------
```swift
    ...
```

Comment:
Could you please check the encoding of this file, I see it here with tons of ^M symbols, like this. Pls, see the attachment

<img width="993" alt="Screen Shot 2021-08-31 at 12 01 24" src="https://user-images.githubusercontent.com/2349286/131483337-f4996436-a1fd-4f3a-9d7c-d2c4bc836e0a.png">